### ISSUE TITLE
Colors in JSX strings do not get highlighted

### ISSUE THREAD

**Issue by gHashTag** [ap/vim-css-color#89]
Why can not the plugin work in Mac Vim version 8.0.946?

**Comment by ap** [comment]
No idea. I use a Mac myself and it works just fine. What does “not work” mean? Do you get errors? How did you install it?

**Comment by gHashTag** [comment]
I`m use iTerm2

`Plug 'ap/vim-css-color'`
:PlugInstall

" Color Settings
syntax enable 
set background=dark
let g:solarized_termcolors=16
colorscheme solarized

![2017-08-24 14 11 16](https://user-images.githubusercontent.com/6774813/29663936-2324516a-88d6-11e7-94fe-b93a68873182.png)

**Comment by ap** [comment]
What does `:set ft?` say in that file?

**Comment by gHashTag** [comment]
syntax=javascript.jsx

**Comment by alex-shamshurin** [comment]
I noticed that the plugin is very slow on macVim, especially in JSX tags

**Comment by ap** [comment]
@alex-shamshurin: Please file a separate issue about that.

**Comment by gHashTag** [comment]
my dotfiles https://github.com/gHashTag/dotfiles

**Comment by gHashTag** [comment]
If I comment the line, then the backlight appears

![2017-10-28 0 51 03](https://user-images.githubusercontent.com/6774813/32126624-4d32e562-bb7a-11e7-8c0a-5fa5ad89ee38.png)

**Comment by DanAndreasson** [comment]
I experience the same issue as gHashTag. Can only see the color if I comment out the line. https://github.com/DanAndreasson/dotfiles

**Comment by ap** [comment]
Well that’s one big hint, eh? Thanks for sticking around and for spotting that – makes my job much easier.

**Comment by g-sam** [comment]
This issue affects colour strings assigned to object keys in any JS file, not just JSX.

**Comment by ap** [comment]
I can’t believe I had this silly typo in there for so long and never realised it nor came around to looking at it myself, that’s embarrassing. Sorry everybody, and thanks to @mimadrid for the {c,p}atch. I merged his ap/vim-css-color#96 a couple hours ago which fixes this issue. Share and enjoy. 😊

**Comment by Fandekasp** [comment]
@ap I just installed your plugin, and am facing the exact same problem, 2 years later.

```vim
Plug 'ap/vim-css-color', { 'for': ['javascript.jsx', 'html', 'htmldjango'] }
```

![](https://i.imgur.com/3x7xz6L.png)
